templates/js/linker.v2.js

```diff
@@ -352,6 +352,8 @@
         options.popupStyles = options.popupStyles || {};
         options.interfaceLang = options.interfaceLang || "english";
         options.contentLang = options.contentLang || "bilingual";
+        options.parentheses = options.parentheses || false;
```

Comment:
Let's rename this parameter `parenthesesOnly` to match `quotationsOnly`. I don't think it needs to be renamed in the python code but it should be renamed here so that people using the linker understand what it means more.